### PR TITLE
feat: command showcase

### PR DESCRIPTION
**Pull request by Olyno**
## What kind of change does this PR introduce?

This pull request introduces the showcase command.

## What is the current behavior?

The #showcase channel on the discord can be improved, and here is one of the improvements possible.

## What is the new behavior?

The showcase command lets users share their project related to Supabase directly on the Discord.

### CODE REVIEW

File: src/commands/showcase.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      .setName('add')
+      .setDescription('Add a new project')
+      .addStringOption(option =>
+        option
+          .setName('name')
+          .setDescription('The name of your project')
+          .setRequired(true)
+      )
+      .addStringOption(option =>
+        option
```
Comment (anand2312) [inline src/commands/showcase.ts:18]: we could use `.setMaxLength` here (and in some of the other fields - maybe project name?) to limit the length of the user input ([doc](https://discordjs.guide/slash-commands/advanced-creation.html#further-validation))

File: src/commands/showcase.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
+import { client } from '../supabase';
+
+export const data = new SlashCommandBuilder()
```
Comment (anand2312) [inline src/commands/showcase.ts:4]: Suggestion: set a cooldown on the command

**Review by anand2312 — CHANGES_REQUESTED** [review]
Looks pretty good!
Another idea - we could use Discord's [modals](https://discordjs.guide/interactions/modals.html) here, the UI would look neater I think. But I'm not sure if validation of the fields is as easy as it is with the slash command. Thoughts?

File: src/commands/showcase.ts (hunk 2 above)
Comment (anand2312) [reply src/commands/showcase.ts:4]: feel free to just dismiss this btw if cooldownds are not necessary. was just a thought 😄

**Comment by Olyno** [comment]
Thanks for your patience. I've just pushed the suggestions you gave me. What do you think? @anand2312

**Review by anand2312 — APPROVED** [review]
LGTM!

**Review by anand2312 — APPROVED** [review]
~~Whoops I clicked comment by accident last time 😅 LGTM again!~~ Huh nvm I clicked right. Dunno why it still says review required @Olyno

**Review by silentworks — APPROVED** [review]

**Pull request merged**